src/Ldap.php

```diff
@@ -52,6 +52,14 @@ class Ldap
      */
     protected $boundUser = false;
 
+    /**
+     * TRUE if a successful bind has been done that needs unbind()
+     * FALSE if a bind has not been attempted, or the bind does not need unbind()
```

Comment:
When does a `bind` not need `unbind`? If I got that correclty this field declares whether an anonymous bind has been done or not. So don't we already have this infoormation in the field `boundUser`?  If `$boundUser` is `NULL` we do not need an unbind.

Reply:
I got rid of this var and used your suggestion. It ends up being equivalent code.